### PR TITLE
web/satellite: tslint update

### PR DESCRIPTION
**Pull request by crawter**
What: More rules added to our tslint config to produce more typed code
No logic was changed, its only import sorting, replacing `const` with `let` and indentations fixed 
Why:

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?
 - [ ] Do the database access patterns make sense?

### CODE REVIEW

File: web/satellite/src/api/apiKeys.ts [hunk 1]
```diff
@@ -68,8 +68,8 @@ export class ApiKeysApiGql extends BaseGql implements ApiKeysApi {
         };
 
         const response = await this.mutate(query, variables);
-        let key: any = response.data.createAPIKey.keyInfo;
-        let secret: string = response.data.createAPIKey.key;
+        const key: any = response.data.createAPIKey.keyInfo;
```
Comment (Barterio) [inline web/satellite/src/api/apiKeys.ts]: Is it still any?

File: web/satellite/tslint.json [hunk 2]
```diff
@@ -12,61 +12,60 @@
...
     "no-invalid-template-strings": true,
-    // Disallows using the this keyword outside of classes.
-    "no-invalid-this": true
+    "no-invalid-this": true,       
+    "no-static-this": true,    
+    "no-var-keyword": true,
+    "newline-before-return": true,
+    "object-literal-sort-keys": false,
+    "one-variable-per-declaration": [true, "ignore-for-loop"],
+    "ordered-imports": false,
```
Comment (Barterio) [inline web/satellite/tslint.json]: ordered imports : true

File: web/satellite/tslint.json [hunk 3]
```diff
@@ -12,61 +12,60 @@
...
+    "eofline": true,
+    "indent": [true, "spaces", 4],    
+    "interface-name": false,
+    "import-spacing": true,
+    "no-async-without-await": true,
+    "no-boolean-literal-compare": true,
+    "no-conditional-assignment": true,
+    "no-consecutive-blank-lines": [true, 1],
     "no-console": [true, "log"],
     "no-default-export": false,
```
Comment (Barterio) [inline web/satellite/tslint.json]: should we disable default export?

Comment (crawter) [reply web/satellite/tslint.json]: it seems that vue components should be default exported

File: web/satellite/src/api/apiKeys.ts (hunk 1 above)
Comment (crawter) [reply web/satellite/src/api/apiKeys.ts]: unfortunatelly

**Pull request merged**